src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("vvrr");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("stop");

```

Comment:
```suggestion
        System.out.println("Bulldozer stop");
```
detail the message to make sure everything is working correctly